theme-contract/src/main/java/com/orange/ods/theme/OdsThemeConfigurationContract.kt

```diff
@@ -46,6 +46,12 @@ interface OdsThemeConfigurationContract : Parcelable {
     val demoGuideline: OdsDemoGuideline
         get() = OdsDemoGuideline(typography)
 
+    /**
+     * Chip type customization
+     * By default the chips are outlined. If your theme needs to use filled chips, set this parameter to false.
+     */
+    val outlinedChips: Boolean
```

Comment:
We could add a `Components` object (which would be the equivalent of `Typography` and `OdsColors` but for components) containing a `chips` enum variable to select the type of chips to use in the theme.